tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -826,4 +859,95 @@ task CalculateContamination {
         Float contamination = read_float("contamination.txt")
         Float contamination_error = read_float("contamination_error.txt")
     }
+}
+
+task SamToFastq {
+  input {
+    File bam
+    String output_prefix
+    String docker = "us.gcr.io/broad-gotc-prod/picard-cloud:2.26.6"
```

Comment:
Might as well go to 2.26.11 (which address the log4j vulnerability.)

Reply:
done